libraries/AP_Mount/AP_Mount.h

```diff
@@ -181,6 +182,7 @@ class AP_Mount
 
     MAV_RESULT handle_command_do_mount_configure(const mavlink_command_long_t &packet);
     MAV_RESULT handle_command_do_mount_control(const mavlink_command_long_t &packet);
+    bool _reached_altitude = false;  // flag to mark that minimum altitude reached to retract the mount
```

Comment:
Please put a blank line before this one so the variables are separated from the methods.

Also, remove the false initialization, it is already false and just increases binary size.